recipes/naive-tsearch/all/conanfile.py

```diff
@@ -1,16 +1,18 @@
-from conans import CMake, ConanFile, tools
+from conan import ConanFile
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 import os
 
+required_conan_version = ">=1.53.0"
 
 class NaiveTsearchConan(ConanFile):
     name = "naive-tsearch"
     description = "A simple tsearch() implementation for platforms without one"
-    topics = ("conan", "naive-tsearch", "tsearch", "search", "tree", "msvc")
     license = "MIT"
-    homepage = "https://github.com/kulp/naive-tsearch"
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kulp/naive-tsearch"
+    topics = ("tsearch", "search", "tree", "msvc")
```

Comment:
```suggestion
    topics = ("tsearch", "search", "tree", "msvc")
    package_type = 'static-library'
```

@toge sorry, after talking to Conan team, they said is better keeping static-library, but add a configure in case of header only